AEPIdentity/Sources/IdentityState.swift

```diff
@@ -164,8 +165,11 @@ class IdentityState {
         identityProperties.mergeAndCleanCustomerIds(customerIds)
         customerIds.removeAll(where: { $0.identifier?.isEmpty ?? true }) // clean all identifiers by removing all that have a nil or empty identifier
 
+        // check is this is forceSync event. Currently the checks for event.forceSync is not very useful as there no events with that property set. It still here to support that workflow when there is a way to update the eventData.
```

Comment:
rephrasing suggestion:
// Checks if this is forceSync event: either this is the first event processed by the extension ad boot time or if the eventData contains the forceSync flag for backwards compatibility.